kmir/src/kmir/kdist/mir-semantics/rt/data.md

```diff
@@ -173,6 +173,8 @@ A variant `#forceSetLocal` is provided for setting the local value without check
 ```k
   syntax KItem ::= #setLocalValue( Place, Evaluation ) [strict(2)]
                  | #forceSetLocal ( Local , Evaluation ) [strict(2)]
+                 | #forceSetPlaceValue ( Place , Evaluation ) [seqstrict(2)]
+                 | #writeProjectionForce ( Value )
```

Comment:
These two constructs are only used by code in `spl-token.md` (`forceSetPlaceValue` is only created there, `writeProjectionForce` is only created from `forceSetPlaceValue`). Maybe the constructs should remain inside `spl-token.md` to not have difference between feature branch and master in `data.md`?